src/Compilers/CSharp/Portable/NullableContextOptions.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    /// <summary>
+    /// Specifies the nullable context.
+    /// </summary>
+    public enum NullableContextOptions : byte
+    {
+        /// <summary>
+        /// Nullable annotation and warning contexts are disabled.
+        /// </summary>
+        Disabled,
+
+        /// <summary>
+        /// Nullable annotation and warning contexts are enabled.
+        /// </summary>
+        Enabled,
```

Comment:
The msbuild property and the command-line and the `#nullable` directive accept "enable", "disable" and "safeonly". But this enum has "Enabled" and "Disabled".
I think the enum should also use "Enable" and "Disable".